### DISCUSSION TITLE
Feature Request: Make the taskbar icons progressively smaller & reduce the space b/w them to the point where they resemble small Quick Link icons to make space for additional icons so there's less need to scroll!

### DISCUSSION THREAD

**Discussion by Droyk** [valinet/ExplorerPatcher#753] [Ideas]
Hey, @valinet Instead of showing arrow keys to find the additional icons. 
![image](https://user-images.githubusercontent.com/15848350/147472575-65e759b5-b061-422c-85bc-3c633dc815e4.png)
Make the taskbar icons progressively smaller & reduce the space b/w them to make space for additional icons.
The scroll arrows appear once the number of taskbar icons hits a certain threshold. 
So instead of that, the taskbar icons should progressively get smaller and smaller to a point where they resemble the **Quick Links.** 
![image](https://user-images.githubusercontent.com/15848350/147473733-1d7e4276-5d48-4422-ade4-366ce5017182.png)
only after then scroll arrows should appear to see the hidden icons.

Here I made a quick video where windows do this though only on the horizontal taskbar and it only progressively reduces the "SPACE" b/w icons.
https://gfycat.com/ifr/gargantuancreamyangelfish/?hd=1&speed=2&controls=0
as you can see in the horizontal taskbar adding more icons will progressively reduce the space b/w each icon.
What I'm requesting is the icons also should get **smaller just as the space b/w them.**
I just hate having to hunt and mess with the scroll arrows just to find a certain taskbar icon, which results in more visual searching and mouse movement. Definitely less ergonomic. & since I place my taskbar on the right side of the monitor (vertical taskbar)
I have to do this thing way often so please make this possible.

**Please make this possible for both Horizontal & Vertical taskbar**

**Comment by Droyk** [comment]
@valinet Would it be possible to implement it? doesn't matter the taskbar 10 or 11 though would appreciate it on both.

**Reply by valinet** [reply]
Unlikely. It's not my taskbar, I don't have the source code of it, disassembling and making sense of it is much harder. And I don't have much time to work on this either.
